From 5d239ed3daa03b7e2a1c65f5d6181592f43aa50b Mon Sep 17 00:00:00 2001
From: Chen Lei <lei.chen@rock-chips.com>
Date: Thu, 13 Dec 2018 16:21:13 +0800
Subject: [PATCH] ARM: dts: rk322x: sdmmc add cd-gpios

This modification is to solve the problem of hot plug failure when
SD card is inserted after boot, because det pin is initialized as GPIO

Change-Id: I1cd49faa92fa7502c23dd30c2c35fa712e5d0ea3
Signed-off-by: Chen Lei <lei.chen@rock-chips.com>
---
 arch/arm/boot/dts/rk322x.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rk322x.dtsi b/arch/arm/boot/dts/rk322x.dtsi
index 9ca100e8288b..eacc1b07f7a7 100644
--- a/arch/arm/boot/dts/rk322x.dtsi
+++ b/arch/arm/boot/dts/rk322x.dtsi
@@ -1087,6 +1087,7 @@
 		fifo-depth = <0x100>;
 		pinctrl-names = "default";
 		pinctrl-0 = <&sdmmc_clk &sdmmc_cmd &sdmmc_bus4>;
+		cd-gpios = <&gpio1 RK_PC1 GPIO_ACTIVE_LOW>;
 		status = "disabled";
 	};
 
-- 
2.35.3

